test/test_round.py

```diff
@@ -179,3 +179,20 @@ def test_round_ints(fi, mldtype):
 
         mlval = _mlround(v, mldtype)
         np.testing.assert_equal(val, mlval)
+
+
+@pytest.mark.parametrize("fi", all_formats, ids=str)
+def test_round_roundtrip(fi):
+    dec = lambda v: decode_float(fi, v).fval
```

Comment:
Unused lambda expression.

Reply:
Thanks, must figure out how to scan for that in pre-commmit.